### DISCUSSION TITLE
Final release for Infinix Inbook Support.EOL.

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#17] [Discuss about the release]
- This is the final release from ME...
- I am giving some files which may support & these files under development.
- These files may help for the people who want to maintain the EFI from now.
- I have patched the USBs for infinix, and file is only needed The USBMap.kext.
- And the EFI you need to work on is the latest version of my Infinix Hackintosh release.
- Don't have a good day! have a great day. 

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/Final'>Final release for Infinix Inbook Support.EOL.</a>.</em>

**Comment by ghost** [comment]
is the latest release gone? why that is?

**Comment by devboloji** [comment]
@bunnybavros yeah, I actually removed the Release due to some severe issues raising in the EFI. So, i will fix the issue and release the Release soon. So, Don't worry about it.

**Comment by ghost** [comment]
By the way, Do you think is it possible to fix the HDMI port since it is the common problem of this coffee lake?
Because I have tried the **UGREEN USB-C 10 in 1 Hub** to connecting my external monitor, I even tried with your old EFIs, because the USB C is not disabled yet and it is still not working.

I am about to get a USB 3.0 to HDMI one, but I am not really sure if it going to fix the problem since my USB-C Dock from UGREEN is not working too.

**Device: Infinix INBook X1 Pro (Core i7 with 16 GBs of RAM)
UGREEN USB-C 10 in 1 Hub that i tried:**  https://www.ugreen.com/products/10-in-1-usb-c-hub

**Reply by devboloji** [reply]
@bunnybavros Our processor name is ICELAKE not coffee lake. I don't think HDMI slot will work in the future. Buy Usb 3.0 to hdmi converter, I already mentioned this topic in my github. Once read it (in status). Many of infinix users used usb3.0 converter. So, 3.0 works.

**Reply by ghost** [reply]
oh sorry! i barely remember about the processor name! haha
btw, I'll try to get one of it and let us see if its going to work! ‘,:)!
